state/testing/watcher.go

```diff
@@ -92,6 +92,22 @@ loop:
 	c.AssertNoChange()
 }
 
+func (c NotifyWatcherC) AssertAtleastOneChange() {
```

Comment:
should be `AssertAtLeastOneChange`, capital L

Reply:
I'll change it in 3.1.